### PR TITLE
MINOR: [JS] Update README to reflect new OmniSci JS Arrow use

### PR DESCRIPTION
**Pull request by jclay**

### CODE REVIEW

File: js/README.md [hunk 1]
```diff
@@ -147,37 +147,33 @@ for (let i = -1, n = column.length; ++i < n;) {
...
-    // Stringify the table to CSV with row numbers
-    table.toString({ index: true }))
-  .subscribe((csvStr) =>
-    console.log(csvStr));
+        FROM flights
+        WHERE dest_city ILIKE 'dallas'
+        LIMIT 5
+`)
+
+console.log(table.toString({ index: true }))
```
Comment (TheNeuralBit) [inline js/README.md:176]: I think it's a little odd to have a usage example for another library here. What do you think about just linking to the mapd/connector documentation and explaining it returns arrow JS objects?

**Review by TheNeuralBit — COMMENTED** [review]
Just one question here

File: js/README.md (hunk 1 above)
Comment (domoritz) [reply js/README.md:176]: Done in https://github.com/apache/arrow/pull/10335. I think we can close this pull request.

**Comment by kou** [comment]
Closed in favor of #10335.

**Pull request closed without merging**